src/dmd/frontend.h

```diff
@@ -970,7 +970,7 @@ struct Visibility
 {
     Kind kind;
     Package* pkg;
-    bool isMoreRestrictiveThan(const Visibility other) const;
```

Comment:
How do we access opCmp from c++?

Reply:
It's not used from C++, as discussed on Slack.